viadot/sources/spectrum.py

```diff
@@ -0,0 +1,166 @@
+from typing import List, Dict, Literal
+import boto3
+
+import pandas as pd
+import awswrangler as wr
+
+
+class Spectrum:
+    """
+    A class for pulling data from and uploading to Spectrum.
+
+    Args:
+        profile_name (str, optional): The name of the profile.
```

Comment:
profile -> AWS profile

Reply:
Updated as per [🎨 updated docstrings and fixed typos](https://github.com/dyvenia/viadot/pull/579/commits/3080223b0461b46ad9260ba51f35b6a41b4a0812)